include/rocksdb/options.h

```diff
@@ -1707,6 +1705,18 @@ struct ReadOptions {
   // Default: empty (every table will be scanned)
   std::function<bool(const TableProperties&)> table_filter;
 
+  // Experimental
+  //
+  // If auto_readahead_size is set to true, it will trim the readahead_size
+  // during scans upto iterate_upper_bound and won't prefetch beyond
+  // iterate_upper_bound.
```

Comment:
Maybe make the wording more generic to leave room for future improvements to the readahead size calculation, like block cache buffering.